stubs/ldap3/ldap3/utils/tls_backport.pyi

```diff
@@ -1 +1,3 @@
 class CertificateError(ValueError): ...
+
+def match_hostname(cert, hostname): ...
```

Comment:
We could make the stub depend on `types-backports.ssl_match_hostname`, and re-export from there.

Reply:
don't know why this shows up twice now